skiko/src/nativeJsMain/cpp/Typeface.cc

```diff
@@ -181,14 +181,10 @@ SKIKO_EXPORT bool org_jetbrains_skia_Typeface__1nGetKerningPairAdjustments
   SkTypeface* instance = reinterpret_cast<SkTypeface*>(ptr);
   if (count > 0) {
       std::vector<int> adjustments(count);
```

Comment:
Looks like a dead code

Reply:
We can not rely on what happens on frontend - in other words if something can happen it will happen and we just need to address any situation in my opinion

Reply:
@Schahen I mean vector `adjustments` is not used, so we can delete this